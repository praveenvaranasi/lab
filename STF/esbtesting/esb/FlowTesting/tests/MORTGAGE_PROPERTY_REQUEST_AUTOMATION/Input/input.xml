<ns1:HTTPRequest xmlns:ns1="http://www.fiorano.com/httpReceive"><Params><loan_requests>51216122</loan_requests><firstname>Tom</firstname><lastname>Davis</lastname><ssn>500111223</ssn><contactno>890-120-123</contactno><email>tom.davis@yahoo.com</email><presentaddress>5th Avenue</presentaddress><city>NY</city><state>NY</state><zipcode>10008</zipcode><prop_address>7th Avenue</prop_address><prop_city>NY</prop_city><prop_state>NY</prop_state><prop_zipcode>10006</prop_zipcode><loan_amount>60000</loan_amount><loan_type>Fixed</loan_type><repay_time>5 years</repay_time></Params><Data/></ns1:HTTPRequest>
