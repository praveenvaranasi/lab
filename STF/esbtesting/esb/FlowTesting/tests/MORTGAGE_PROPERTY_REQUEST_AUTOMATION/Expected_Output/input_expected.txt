<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
"http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Mortgage - Property Information</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<style type="text/css">
<!--
body {
 margin-left: 0px;
 margin-top: 0px;
 margin-right: 0px;
 margin-bottom: 0px;
}
.stdtext {  font-family: Arial, Geneva, Helvetica, "Lucida Sans", sans-serif; font-size: 9pt;line-height: 15px; color: #000000; text-decoration: none}
.itemBorder { border:1px solid #999999 }
.pn {  font-family: Arial, Geneva, Helvetica, "Lucida Sans", sans-serif; font-size: 9pt;line-height: 20px; color: #000000; text-decoration: none}

.smalltext {  font-family: Arial, Geneva, Helvetica, "Lucida Sans", sans-serif; font-size: 9pt;line-height: 15px; color: #000000; text-decoration: none}
.verysmalltext {  font-family: Arial, Geneva, Helvetica, "Lucida Sans", sans-serif; font-size: 7pt;line-height: 13px; color: #003366}
.devtext {  font-family: Arial, Helvetica, sans-serif; font-size: 9pt;line-height: 15px; color: #000000; text-decoration: none}




.smalltext_news {  font-family: Arial,  Geneva, Helvetica, "Lucida Sans", sans-serif; font-size: 8pt;line-height: 12px; color: #000099 ; font-style: italic; text-decoration: none}

body {  background : #FFFFFF; } 
a { color: #000099;  text-decoration: none}
a:active  { color: #065590 }


h1 { font-family: "Arial Narrow", Arial, Helvetica, sans-serif; font-size: 40px; line-height: 32px; font-weight: bold; color: #cccccc; }
h2 {  font-family: Arial, Helvetica, sans-serif; font-weight: bold; font-size: 10pt; color: #000000}
h3 { font-family: Arial, Helvetica, sans-serif; font-size: 12pt; line-height: 16pt ; color: #000000; font-weight: normal}

ul {  list-style-image:  url('/images/listbullet.gif'); list-style-type: square}
ul.navlist { list-style-image: url('/images/navbullet.gif'); list-style-type: square; font-family: Arial, Helvetica, sans-serif; font-size: 9pt; margin-top:0px; margin-bottom:0px;  }
.bgconnector {  }

td {  font-family: Arial, Helvetica, sans-serif; font-size: 9pt; line-height: 16pt; color: #000000}
td.nolh {  line-height: normal}
td.navtopbg { font-family: Arial, Helvetica, sans-serif; color: #FFFFFF; font-weight: bold; background-color: #00408a; background-image: url(/images/navtop.gif)}
td.pname { font-family: "Arial Narrow", Ariel, Helvetica, sans-serif; font-size: 32px; font-weight: lighter; color: #065590; padding-bottom: 6px}
td.picture {  text-align: right; vertical-align: top}
td.content {  font-family: Arial, Helvetica, sans-serif; font-size: 10pt; line-height: 16pt; color: #000000}
td.contenttop { font-family: Arial, Helvetica, sans-serif; font-size: 10pt}
td.rcol {  text-align: center}
td.linel { background-image:  url('/images/vertical.gif')}

.searcharea {  text-align: right; vertical-align: middle}
.press { font-family: "Arial Narrow", Arial, Helvetica, sans-serif; font-size: 18px; font-weight: lighter; color: #065590; }
.presssub {  font-family: Arial, Helvetica, sans-serif; font-weight: bold; font-size: 10pt; color: #065590}
.fineprnt { font-family: Arial, Helvetica, sans-serif; font-size: 8pt;}
.footer { font-family: Arial, Helvetica, sans-serif; font-size: 8pt; text-align: center; color: #666666}
.rquote {  font-size: 10pt; font-style: normal; color: #999933; ; font-weight: 300; font-variant: normal}
.rquoteauthor {  font-size: 10pt; font-style: normal; color: #999999; ; font-weight: normal; font-variant: normal}
.rquotebracket { background-image: url('/images/quote_line.gif')}

.boldblack {  font-weight: bold; color: #000000}
.black {  color: #000000}
.required { font-family: Arial, Helvetica, sans-serif; font-size: 10pt; color: #666666; font-weight: bold}



.navtop { font-family: Arial, Helvetica, sans-serif; color: #FFFFFF; font-weight: bold; }

.navtop { font-family: Arial, Helvetica, sans-serif; color: #FFFFFF; font-weight: bold; }

a.navtopon { font-family: Arial, Helvetica, sans-serif; color: #ffbb00; font-weight: bold}

// .nav {  font-family: Arial, Helvetica, sans-serif; font-weight: bold; color: #00408a; font-size: 10pt; line-height: 14pt }

// a.navon { color: #00408a ; font-family: Arial, Helvetica, sans-serif; font-size: 9pt}

// a.navsub {color: #00408a ; font-family: Arial, Helvetica, sans-serif; font-size: 9pt }

// a.navsubon { color: #B0A640 ; font-family: Arial, Helvetica, sans-serif; font-size: 9pt}
.nav {  font-family: Arial, Helvetica, sans-serif; font-weight: bold; color: #00408a; font-size: 10pt; line-height: normal }

a.navon {  color: #00408a ; font-family: Arial, Helvetica, sans-serif; font-size: 9pt; line-height: normal }

a.navsub {color: #00408a ; font-family: Arial, Helvetica, sans-serif; font-size: 9pt; line-height: normal}

a.navsubon { color: #B0A640 ; font-family: Arial, Helvetica, sans-serif; font-size: 9pt; line-height: normal}

a.navsub2 {color: #00408a; margin-left: 3px; }
a.navsub2on { color: #B0A640 margin-left: 3px;}
a.navfooter { font-family: Arial, Helvetica, sans-serif; color: #065590}
a.navital{color: #00408a ; font-family: Arial, Helvetica, sans-serif; font-size: 8pt; font-style: italic; line-height: normal}
//a:hover { color:#ffbb00; text-decoration:none } /* this will override all other hover definitions */
a:hover { color:#cc0000; text-decoration:none } /* this will override all other hover definitions */
devzone_rhs { color:#000000; text-decoration:none }

//a.navtop:hover { color:#fe9f34; text-decoration:none }
//a.nav:hover { color:#fe9f34; text-decoration:none }
//a.navsub:hover { color:#fe9f34; text-decoration:none }
//a.navfooter:hover { color:#fe9f34; text-decoration:none }

a.navtop:hover { color:#cc0000; text-decoration:none }
a.nav:hover { color:#cc0000; text-decoration:none }
a.navsub:hover { color:#cc0000; text-decoration:none }
a.navfooter:hover { color:#cc0000; text-decoration:none }

a.white_hdr:hover { color:#ffffff; text-decoration:none }


.fi_headline {
font-size: 11pt;
font-family:  verdana, Arial, Helvetica,  sans-serif;
text-align: left;
color: #000099;
font-weight: bold;
font-style: normal;
}

.fi_subhead {
font-size: 9pt;
font-family: verdana, Arial, Helvetica,  sans-serif;
color: #000099;
font-weight: bold;
font-style: normal;
}

.fi_centerhead {
font-size: 9pt;
font-family: Arial, Helvetica,  sans-serif;
text-align: center;
color: #000099;
font-weight: bold;
font-style: normal;
}

.fi_tablehead {
font-size: 10pt;
font-family: Arial, Helvetica,  sans-serif;
text-align: center;
color: #FFFFFF;
font-weight: bold;
font-style: normal;
}

.fi_figure {
font-size: 9pt;
font-family: verdana, Arial, Helvetica,  sans-serif;
text-align: center;
color: #993300;
font-weight: bold;
font-style: normal;
}

.fi_highlight {
font-size: 9pt;
font-family: Arial, Helvetica,  sans-serif;
text-align: left;
color: #cc0000;
font-weight: bold;
font-style: normal;
}

p {  font-family: Arial, Helvetica, sans-serif; font-size: 9pt; line-height: 13pt; color: #000000 ; text-align: justify}

.newpara {  font-family: Arial, Helvetica, sans-serif; font-size: 9pt; line-height: 13pt; color: #000000}

pul {  font-family: Arial, Helvetica, sans-serif; font-size: 9pt; line-height: 12pt; color: #000000}



.white_hdr {  font-family:  verdana, Arial, Helvetica, sans-serif; font-size: 12px; font-weight: bold; color: #FFFFFF}

.whiteTxt {  font-family:  verdana, Arial, Helvetica, sans-serif; font-size: 10px; font-weight: normal; color: #FFFFFF}

.whiteTxtCopy { font-family:  verdana, Arial, Helvetica, sans-serif; font-size: 10px; font-weight: normal; color: #003366}

.white_hdrCopy { font-family:  verdana, Arial, Helvetica, sans-serif; font-size: 12px; font-weight: bold; color: #003366}
.bodytxt {
 font-family: Arial, Helvetica, sans-serif;
 font-size: 10pt;
 color: #666666;
 line-height: 16px;
 
}
TD.content {
 FONT-SIZE: 10pt; COLOR: #666666; LINE-HEIGHT: 12pt; FONT-FAMILY: Arial, Helvetica, sans-serif
}
.sidelinks {
 font-family: Arial, Helvetica, sans-serif;
 font-size: 12px;
}
.bullet {
 font-family: Arial, Helvetica, sans-serif;
 font-size: 9pt;
 color: 61849C;
 font-weight: bold;
}
.newshdr {

font-size: 10pt;
font-family:  Arial, Helvetica,  sans-serif;
color: #000099;
font-weight: bold;
font-style: normal;
}
.txtbox {
 font-family: Arial, Helvetica, sans-serif;
 font-size: 11px;
 color: #003366;
 background-color: #E6F2FF;
}
.body {
 font-family: Arial, Helvetica, sans-serif;
 font-size: 9pt;
 color: #000000;
 line-height: 12pt;
}

.orangehdr {
 font-family: "Arial Narrow";
 font-size: 24px;
 font-weight: bold;
 color: #CC3300;
}
.orangesubhead {
 font-family: Arial, Helvetica, sans-serif;
 font-size: 12px;
 font-weight: bold;
 color: #CC3300;
}
.orangetxt {

 font-family: Arial, Helvetica, sans-serif;
 font-size: 12px;
 font-weight: normal;
 color: #CC3300;
}
.jp {
 font-family: "MS PGothic";
 font-size: 9pt;
 color: #000000;
 text-align: justify;


}
.jpnavon {
 color: #00408a;
 font-family: "MS PGothic";
 font-size: 9pt;
 line-height: normal;
 font-weight: bold;

}
.jpnavsub {

 color: #00408a;
 font-family: "MS PGothic";
 font-size: 9pt;
 line-height: normal;
 font-weight: normal;
}
.jpsubhead {

font-size: 9pt;
font-family: verdana, Arial, Helvetica,  sans-serif;
color: #000099;
font-weight: bold;
font-style: normal;
}
.jporangehdr {

 font-family: "MS PGothic";
 font-size: 24px;
 font-weight: bold;
 color: #CC3300;
}

-->
</style>
<link href="fiorano.css" rel="stylesheet" type="text/css">
</head>

<body>
<br>
<table width="90%"  border="0" align="center" cellpadding="1" cellspacing="0">
  <tr>
    <td bgcolor="#006699"><table width="100%"  border="0" cellpadding="3" cellspacing="0" bgcolor="#CCCCCC">
        <tr>
          <td><table width="100%"  border="0" cellpadding="4" cellspacing="4" bgcolor="#FFFFFF">
              <tr>
                <td colspan="2" class="orangesubhead">Loan Request - Mortgage Property Information</td>
              </tr>
 <tr bgcolor="f3f3f3">
    <td width="36%"><b>Loan Request Id</b></td>
    <td width="64%">51216122</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>Property Address</b></td>
    <td width="64%">7th Avenue</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>City</b></td>
    <td width="64%">NY</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>State</b></td>
    <td width="64%">NY</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>Whether Already Mortgaged</b></td>
    <td width="64%">N</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>Whether Open for Selling </b></td>
    <td width="64%">Y</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>Total Estimated Cost</b></td>
    <td width="64%">120000</td>
     </tr><tr bgcolor="f3f3f3">
    <td width="36%"><b>Last Tax Paid Date </b></td>
    <td width="64%">2005-05-19 11:11:10.0</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>Due Date</b></td>
    <td width="64%">2006-05-18 11:11:10.0</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>Last Amount Paid</b></td>
    <td width="64%">1800</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>Total Amount Due</b></td>
    <td width="64%">0</td>
     </tr> <tr bgcolor="f3f3f3">
    <td width="36%"><b>Remarks </b></td>
    <td width="64%"></td>
     </tr></table></td>
        </tr>
    </table></td>
  </tr>
</table>
<br>

</body>
</html>
